src/App/Fossa/VPS/Scan/RunWiggins.hs

```diff
@@ -43,6 +46,17 @@ data WigginsOpts = WigginsOpts
   , spectrometerArgs :: [Text]
   }
 
+data PathFilters = PathFilters
+  { onlyPaths :: [Path Rel Dir]
+  , excludePaths :: [Path Rel Dir]
+  }
+
+toText :: Path a b -> Text
+toText = T.pack . toFilePath
```

Comment:
This is already defined in `Data.String.Conversion`

Reply:
permalink for reference: https://github.com/fossas/spectrometer/blob/f2f56bea9a9445bb42ad4938f56564553e497879/src/Data/String/Conversion.hs#L75-L76

Reply:
Oh 🤦‍♀️ thanks!

Reply:
Yeah, that module is sort of new, I'm trying to help focus all of our string conversion to that module.